src/app/services/select.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
```

Comment:
For a service that's this small, it's probably better to look into route parameters in angular. Since you're just passing an id to the component on a specific route, you can pass that using a route parameter. 

https://angular.io/guide/router#route-parameters

And then use this to access that route parameter without using observables: 
https://angular.io/guide/router#snapshot-the-no-observable-alternative